### ISSUE TITLE
AC Tool UI doesn't appear in v2.6 or later Uncaught SlingException

### ISSUE THREAD

**Issue by davidrmckinnon** [Netcentric/accesscontroltool#521]
We're currently using v2.5.5 on AEM 6.5.6, but when testing v2.6 or later, the AC Tool UI at http://localhost:4502/mnt/overlay/netcentric/actool/content/overview.html/actool appears empty.

We get this error with v2.7.0
```
2020-11-30 09:10:35.842 DEBUG [org.acac.core.filters.LoggingFilter] request for /mnt/overlay/netcentric/actool/content/overview, with selector null
2020-11-30 09:10:35.842 DEBUG [org.acac.core.filters.LoggingFilter] request for /mnt/overlay/netcentric/actool/content/overview, with selector null
30.11.2020 09:10:36.432 *ERROR* [] [0:0:0:0:0:0:0:1 [1606687835838] GET /mnt/overlay/netcentric/actool/content/overview.html/actool HTTP/1.1] org.apache.sling.engine.impl.SlingRequestProcessorImpl - service: Uncaught SlingException
	at biz.netcentric.cq.tools.actool.ui.AcToolUiService$RequestParameters.getParam(AcToolUiService.java:349)
	at biz.netcentric.cq.tools.actool.ui.AcToolUiService$RequestParameters.fromRequest(AcToolUiService.java:308)
	at biz.netcentric.cq.tools.actool.ui.AcToolUiService.renderUi(AcToolUiService.java:102)
	at biz.netcentric.cq.tools.actool.ui.AcToolUiService.doGet(AcToolUiService.java:68)
	at biz.netcentric.cq.tools.actool.ui.AcToolTouchUiServlet.doGet(AcToolTouchUiServlet.java:55)
30.11.2020 09:10:36.437 *INFO* [] [0:0:0:0:0:0:0:1 [1606687835838] GET /mnt/overlay/netcentric/actool/content/overview.html/actool HTTP/1.1] apps.sling.servlet.errorhandler.Throwable$jsp - redirectResource : null
30.11.2020 09:10:36.438 *ERROR* [] [0:0:0:0:0:0:0:1 [1606687835838] GET /mnt/overlay/netcentric/actool/content/overview.html/actool HTTP/1.1] org.apache.sling.servlets.resolver.internal.SlingServletResolver - Calling the error handler resulted in an error
30.11.2020 09:10:36.438 *ERROR* [] [0:0:0:0:0:0:0:1 [1606687835838] GET /mnt/overlay/netcentric/actool/content/overview.html/actool HTTP/1.1] org.apache.sling.servlets.resolver.internal.SlingServletResolver - Original error class java.lang.NullPointerException
	at biz.netcentric.cq.tools.actool.ui.AcToolUiService$RequestParameters.getParam(AcToolUiService.java:349)
	at biz.netcentric.cq.tools.actool.ui.AcToolUiService$RequestParameters.fromRequest(AcToolUiService.java:308)
	at biz.netcentric.cq.tools.actool.ui.AcToolUiService.renderUi(AcToolUiService.java:102)
	at biz.netcentric.cq.tools.actool.ui.AcToolUiService.doGet(AcToolUiService.java:68)
	at biz.netcentric.cq.tools.actool.ui.AcToolTouchUiServlet.doGet(AcToolTouchUiServlet.java:55)
30.11.2020 09:10:36.439 *ERROR* [] [0:0:0:0:0:0:0:1 [1606687835838] GET /mnt/overlay/netcentric/actool/content/overview.html/actool HTTP/1.1] org.apache.sling.engine.impl.DefaultErrorHandler - Error handler failed:java.io.IOException
30.11.2020 09:10:36.439 *ERROR* [] [0:0:0:0:0:0:0:1 [1606687835838] GET /mnt/overlay/netcentric/actool/content/overview.html/actool HTTP/1.1] org.apache.sling.engine.impl.DefaultErrorHandler - handleError: Response already committed; cannot send error 500java.lang.NullPointerException
```

**Comment by joestieg** [comment]
This is the same issue as #525.

**Comment by kwin** [comment]
This seems to be a regression introduced by https://github.com/Netcentric/accesscontroltool/commit/85f2eb4a13dae44163daad94322d31373992d0a6.
@ghenzler Can you look into that?

**Comment by joestieg** [comment]
We were able to fix this and make it work by just making one line of change in the code and rebuilding it.

We changed the code as shown below in AcToolUiService. (deleted _result = defaultValue;_)

```
    static String getParam(final HttpServletRequest req, final String name, final String defaultValue) {
            String result = req.getParameter(name);
            if (result == null) {
                result = defaultValue;
                return defaultValue; 
            }
           
            return result.trim();

        }
```

**Comment by ghenzler** [comment]
@davidrmckinnon This is fixed in v2.7.1 that is now available
